prow/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 The Jetstack contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: update-config
+update-config:
+	kubectl create configmap config --from-file=config.yaml=config.yaml --dry-run=client -o yaml | kubectl replace configmap config -f -
+
+.PHONY: update-plugins
+update-plugins:
+	kubectl create configmap plugins --from-file=plugins.yaml=plugins.yaml --dry-run=client -o yaml | kubectl replace configmap plugins -f -
+
```

Comment:
I assume this is handy when you want to test a config change without using git?
Do I add a comment or do I remove these targets?

Reply:
tbh, it is going to be inconsistent in either way. My main concern is that folks will think that this is how config changes should be applied. A warning that this is just for local testing and any applied changes should be either reset or pushed to Git shoudl be ok 👍🏼
 and perhaps in future we can make it so that the way Prow and Prow config gets applied is more consistent instead of one from dev laptop, the other from Git

Reply:
done